docs/components/info-unit-groups-image-text.md

```diff
@@ -19,14 +19,17 @@ description: >-
   at:
 
 
-  * https://cfpb.github.io/design-manual/page-components/25-75.html
+  *
+  [https://cfpb.github.io/design-manual/page-components/25-75.html](https://cfpb.github.io/design-manual/page-components/25-75.html)
 
-  * https://cfpb.github.io/design-manual/page-components/50-50.html
+  *
+  [https://cfpb.github.io/design-manual/page-components/50-50.html](https://cfpb.github.io/design-manual/page-components/50-50.html)
 
   *
-  https://cfpb.github.io/capital-framework/components/cf-layout/#custom-content-layouts
+  [https://cfpb.github.io/capital-framework/components/cf-layout/#custom-content-layouts](https://cfpb.github.io/capital-framework/components/cf-layout/#custom-content-layouts)
```

Comment:
We should probably remove links to CF documentation entirely, since they are deprecated in favor of the design system, but since those docs are still up this is okay for now.